src/client/auth.js

```diff
@@ -1,16 +1,67 @@
 const UUID = require('uuid-1345')
 const yggdrasil = require('yggdrasil')
+const fs = require('mz/fs')
+const promisify = require('es6-promisify')
```

Comment:
There is already a promisify function in node https://nodejs.org/api/util.html#util_util_promisify_original

Reply:
Alright. Again, I just mirrored this behavior from the minecraft wrapper module but it should be fine either way